CODE_OF_CONDUCT.md

```diff
@@ -67,7 +67,7 @@ members of the project's leadership.
 
 ### Appeal
 
-If you are banned you may contest the decision. To do so email conduct@angular.io with the subject line "Repeal Ban for {{your name here}}" and body with the responses to the following:
+If you are banned you may contest the decision. To do so email conduct@angular.dev with the subject line "Repeal Ban for {{your name here}}" and body with the responses to the following:
```

Comment:
@josephperrott, does this email exist?

Reply:
It does not

Reply:
Oop, okie dokie lemmie revert that back real quick and it can be addressed once it gets created.